implementation/.github/workflows/auto-merge.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         echo "::set-output name=mergeable_state::$(echo "${payload}" | jq -r -c .mergeable_state)"
 
     - name: Merge
-      if: ${{ steps.pull_request.outputs.mergeable_state == 'clean' }}
+      if: ${{ steps.pull_request.outputs.mergeable_state == 'clean' || steps.pull_request.outputs.mergeable_state == 'unstable' }}
```

Comment:
are there cases where 'clean' can still happen?

Reply:
I think the issue here is that the GitHub API is not consistent in what it returns. I believe there is some sort of race condition wherein sometimes it will return `clean` and other times it returns `unstable`. Both should be acceptable.

Reply:
ahhh ok that's fair then